index.html

```diff
@@ -69,7 +83,7 @@ <h2 class="title">
 					<div class="category_description">
 						<h3 class="category_title">Register</h3>
 						<h2 class="title">
-							판매를 원하는<br />
+							<span>판매를 원하는 </span>
```

Comment:
❗️ 수정요청
지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
							<span>판매를 원하는</span>
```